app/src/main/res/layout/contract_detail_row.xml

```diff
@@ -23,6 +23,7 @@
         android:layout_height="wrap_content"
         android:textAlignment="textEnd"
         android:textAppearance="?textAppearanceBody1"
+        android:textColor="?colorPrimary"
```

Comment:
We should use a correctly predefined style. We should challenge this in the designs.